### ISSUE TITLE
Failed to create user with permissions using CLI

### ISSUE THREAD

**Issue by victorlmh** [influxdata/influx-cli#342]
After running influxdb (version 2.1.1) and trying to create a user with read and write permission to some buckets using CLI (2.2.1) (https://docs.influxdata.com/influxdb/cloud/reference/cli/influx/v1/auth/create/), the created user does not have the necessary permissions.

![error](https://user-images.githubusercontent.com/1843456/146260780-fa885e76-1b17-4edf-bd56-56238f2be1b1.png)

command executed: influx v1 auth create --username github --password github_pw --token token_influxdb --read-bucket 742a8eb3969cdf13 --read-bucket 742a8eb3969cdf13 --read-bucket fdc924dc99d5b18f --write-bucket fdc924dc99d5b18f

Step by step:

1- Create a database with more than one bucket.
2- Check the ID for each bucket
3- Create an user using the CLI ((https://docs.influxdata.com/influxdb/cloud/reference/cli/influx/v1/auth/create/)) for more than one bucket
4- Check the permissions for the created user

**Comment by jasonf20** [comment]
Same thing is happening to me

**Comment by BenoitPoulet** [comment]
Hello,

Same here, i tried to create V1 Auth on severals buckets, but as result i get only rights on 1 bucket.
The same bucket ID is used on all the authorizations.

```
0 ✓ dirv-monitoring-01 ~ $ influx v1 auth create --read-bucket 60c121c84e72075d --write-bucket 60c121c84e72075d --read-bucket 4e70e654e7ae6274 --write-bucket 4e70e654e7ae6274 --read-bucket f57e67e2c0fa7e99 --write-bucket f57e67e2c0fa7e99 --read-bucket 7205ace3caaf1fdc --write-bucket 7205ace3caaf1fdc --read-bucket ff0a229d66f95bc1 --write-bucket ff0a229d66f95bc1 --read-bucket abb2be08751b6946 --write-bucket abb2be08751b6946 --read-bucket 1980a638df195248 --write-bucket 1980a638df195248 --read-bucket 92c7ccb316c6c279 --write-bucket 92c7ccb316c6c279 --read-bucket cb931d2dc8c34b33 --write-bucket cb931d2dc8c34b33 --read-bucket cb0c59c3c7d096f5 --write-bucket cb0c59c3c7d096f5 --read-bucket 3d2b52185f0ae637 --write-bucket 3d2b52185f0ae637 --read-bucket 68ff4433488e24e2 --write-bucket 68ff4433488e24e2 --read-bucket f929e76fc37151d6 --write-bucket f929e76fc37151d6  --username metrics --password 'xxxxxxxxxxx'
ID                      Description     Username        v2 User Name    v2 User ID              Permissions
08ac2d5dd2d44000                        metrics         admin           089a3d246c944000        [read:orgs/920ba989433e2dc2/buckets/f929e76fc37151d6 read:orgs/920ba989433e2dc2/buckets/f929e76fc37151d6 read:orgs/920ba989433e2dc2/buckets/f929e76fc37151d6 read:orgs/920ba989433e2dc2/buckets/f929e76fc37151d6 read:orgs/920ba989433e2dc2/buckets/f929e76fc37151d6 read:orgs/920ba989433e2dc2/buckets/f929e76fc37151d6 read:orgs/920ba989433e2dc2/buckets/f929e76fc37151d6 read:orgs/920ba989433e2dc2/buckets/f929e76fc37151d6 read:orgs/920ba989433e2dc2/buckets/f929e76fc37151d6 read:orgs/920ba989433e2dc2/buckets/f929e76fc37151d6 read:orgs/920ba989433e2dc2/buckets/f929e76fc37151d6 read:orgs/920ba989433e2dc2/buckets/f929e76fc37151d6 read:orgs/920ba989433e2dc2/buckets/f929e76fc37151d6 write:orgs/920ba989433e2dc2/buckets/f929e76fc37151d6 write:orgs/920ba989433e2dc2/buckets/f929e76fc37151d6 write:orgs/920ba989433e2dc2/buckets/f929e76fc37151d6 write:orgs/920ba989433e2dc2/buckets/f929e76fc37151d6 write:orgs/920ba989433e2dc2/buckets/f929e76fc37151d6 write:orgs/920ba989433e2dc2/buckets/f929e76fc37151d6 write:orgs/920ba989433e2dc2/buckets/f929e76fc37151d6 write:orgs/920ba989433e2dc2/buckets/f929e76fc37151d6 write:orgs/920ba989433e2dc2/buckets/f929e76fc37151d6 write:orgs/920ba989433e2dc2/buckets/f929e76fc37151d6 write:orgs/920ba989433e2dc2/buckets/f929e76fc37151d6 write:orgs/920ba989433e2dc2/buckets/f929e76fc37151d6 write:orgs/920ba989433e2dc2/buckets/f929e76fc37151d6]

0 ✓ dirv-monitoring-01 ~ $ influx v1 auth ls
ID                      Description     Username        v2 User Name    v2 User ID              Permissions
08ac2d5dd2d44000                        metrics         admin           089a3d246c944000        [read:orgs/920ba989433e2dc2/buckets/f929e76fc37151d6 read:orgs/920ba989433e2dc2/buckets/f929e76fc37151d6 read:orgs/920ba989433e2dc2/buckets/f929e76fc37151d6 read:orgs/920ba989433e2dc2/buckets/f929e76fc37151d6 read:orgs/920ba989433e2dc2/buckets/f929e76fc37151d6 read:orgs/920ba989433e2dc2/buckets/f929e76fc37151d6 read:orgs/920ba989433e2dc2/buckets/f929e76fc37151d6 read:orgs/920ba989433e2dc2/buckets/f929e76fc37151d6 read:orgs/920ba989433e2dc2/buckets/f929e76fc37151d6 read:orgs/920ba989433e2dc2/buckets/f929e76fc37151d6 read:orgs/920ba989433e2dc2/buckets/f929e76fc37151d6 read:orgs/920ba989433e2dc2/buckets/f929e76fc37151d6 read:orgs/920ba989433e2dc2/buckets/f929e76fc37151d6 write:orgs/920ba989433e2dc2/buckets/f929e76fc37151d6 write:orgs/920ba989433e2dc2/buckets/f929e76fc37151d6 write:orgs/920ba989433e2dc2/buckets/f929e76fc37151d6 write:orgs/920ba989433e2dc2/buckets/f929e76fc37151d6 write:orgs/920ba989433e2dc2/buckets/f929e76fc37151d6 write:orgs/920ba989433e2dc2/buckets/f929e76fc37151d6 write:orgs/920ba989433e2dc2/buckets/f929e76fc37151d6 write:orgs/920ba989433e2dc2/buckets/f929e76fc37151d6 write:orgs/920ba989433e2dc2/buckets/f929e76fc37151d6 write:orgs/920ba989433e2dc2/buckets/f929e76fc37151d6 write:orgs/920ba989433e2dc2/buckets/f929e76fc37151d6 write:orgs/920ba989433e2dc2/buckets/f929e76fc37151d6 write:orgs/920ba989433e2dc2/buckets/f929e76fc37151d6]

```

```
0 ✓ dirv-monitoring-01 ~ $ influx version
Influx CLI 2.2.1 (git: 31ac783) build_date: 2021-11-09T21:24:22Z
```

**Comment by BenoitPoulet** [comment]
Seems the same as : https://github.com/influxdata/influx-cli/pull/328

**Comment by victorlmh** [comment]
@BenoitPoulet, yes. 

Unfortunately, there is no release version available with the bug fix. The commit is in the main. I am waitting the release version.

**Comment by gungorerhan** [comment]
Is there a workaround that we can use until the new release?

**Comment by DStrand1** [comment]
Fixed and released in InfluxCLI 2.3.0
